src/networking.c

```diff
@@ -209,6 +210,29 @@ cc_oci_tap_create(const gchar *const tap) {
 	return ret;
 }
 
+gboolean
+is_interface_ovs(struct cc_oci_net_if_cfg* if_cfg) {
```

Comment:
Can you add a comment about our current dummy interface method of creating a stand in dummy interface for the ovs interface. Longer term we need to figure out a better method of handling ovs interfaces.